app/enterprise/k8s-changelog.md

```diff
@@ -1,126 +1,4 @@
 ---
 title: Kong Enterprise k8s Changelog
+redirect_to: https://github.com/Kong/kubernetes-ingress-controller/blob/main/CHANGELOG.md
 ---
-
```

Comment:
No, we wish to keep this page around. It just doesn't need to be linked to from anywhere.

Reply:
Ack - I thought this was versioned and would stay in older pages. Fixing.

Reply:
Adding a "Kong Enterprise k8s is deprecated" note to the top of this file. What date should I put for this? The KE 2.1.3.0 GA date?

Reply:
> What date should I put for this? The KE 2.1.3.0 GA date?

Yes